1-estrategia/README.md

```diff
@@ -2,22 +2,129 @@
+
+* Intuitividad: Otro de los propósitos de la plataforma web es que el usuario pueda vivir una experiencia agradable y sin frustraciones, esto se logra mediante técnicas de navegación que sean claras, reduciendo la ambigüedad en la interpretación, es decir, si lo que busca es información sobre la documentación, encontrará una pestaña que diga “Trámite de documentos”, y que al clickear salga efectivamente la información sobre dónde debe ir, qué requisitos cumplimentar, las entidades a las cuales puede contactar. Un error que le genere frustración al usuario sería que al momento de clickear en la pestaña en lugar de encontrar esa información encuentre micro historias de las experiencias de otros migrantes en la ciudad.
+
+* Credibilidad: La credibilidad en nuestro sitio se debe sostener en parte a partir del trabajo ético, transparente, moral y profesional de todos los que forman parte de este sitio web, la otra parte, vendrá dado gracias a las alianzas con instituciones del orden público y reconocidas ONGs. Presentaremos de manera visible, una sección dedicada a los Partners, que serán justamente estas instituciones y organizaciones con quienes trabajamos conjuntamente. Todo esto ayudará en el tiempo a fortalecer la confianza, fiabilidad y credibilidad en nuestro sitio web.
+
 ### Indicadores de éxito
 
+* Happiness (Satisfacción de los usuarios en relación con el producto): Se quiere la satisfacción del usuario mediante una valoración que realizarán ellos mismos al momento de  abandonar la plataforma. Es decir un tipo de encuesta donde 1 es insatisfecho y 5 es muy satisfecho + un box de sugerencias.
+
+* Engagement (Grado de respuesta del usuario sobre el producto): “Para medir el grado de recomendación que realizan los usuarios de nuestro sitio web podemos utilizar el criterio de clasificación denominado NPS (Net Promoter Score), para lo cual se clasifican en tres grupos: Los detractores serían aquellos con una puntuación de entre 0 y 6, los pasivos aquellos que nos otorguen entre un 7 y un 8 y los promotores serían aquellos que puntuaron entre un 9 y un 10 dicha sección de nuestra web”.→ Para nuestro sitio, podemos utilizar este criterio de clasificación para medir el grado de recomendación de unos inmigrantes a otros. Es muy interesante de utilizar este NPS para nuestro sitio porque podemos saber los sentimientos que tienen las personas por el sitio web.
```

Comment:
Fuente de este contenido. ¿Cómo vais a recoger esta puntuación? ¿Qué es exactamente lo que están puntuando? Creo que entiendo a dónde queréis llegar, pero este párrafo es muy confuso.